_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,249 @@
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+<a href="/test-assets/59br37/poem.html">Full text of the poem</a>
+```
+
+#### Passed Example 4
+
+This [text node][] is not [clipped][] with `overflow: hidden` because it has a parent with `overflow: auto` at a [viewport size][] of 640 by 512.
+
+```html
+<div style="overflow: hidden; height: 2em; font-size: 16px;">
+	<div style="overflow: auto; height: 2em;">
+		Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+		lore. While I nodded, nearly napping, suddenly there came a tapping.
+	</div>
+</div>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This [text node][] is [clipped][] because it has a fixed height that does not leave enough space for the content to wrap.
+
+```html
+<div style="overflow: hidden; height: 1.5em; font-size: 16px;">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+```
+
+#### Failed Example 2
+
+This [text node][] is [clipped][] because its height is relative to the viewport height, leaving insufficient space for the page to be zoomed to 200% at a [viewport size][] of 1280 by 1024.
+
+```html
+<div style="overflow: hidden; height: 16vh; font-size: 16px;">
+	Once upon a midnight dreary, while I pondered, weak and weary,<br />
+	Over many a quaint and curious volume of forgotten lore.<br />
+	While I nodded, nearly napping, suddenly there came a tapping,<br />
+	As of some one gently rapping, rapping at my chamber door.<br />
+	“’Tis some visitor,” I muttered, “tapping at my chamber door.<br />
+	Only this and nothing more.”
+</div>
+```
+
+#### Failed Example 3
+
+This [text node][] is [clipped][] by style that is applied at a [viewport size][] width of 640.
+
+```html
+<style>
+	@media screen and (max-width: 640px) {
+		.myContainer {
+			height: 1.5em;
+			width: 50%;
+			overflow: hidden;
+			font-size: 16px;
+		}
+	}
+</style>
+<div class="myContainer">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping, As of some one gently rapping, rapping at my
+	chamber door. “’Tis some visitor,” I muttered, “tapping at my chamber door. Only this and nothing more.”
+</div>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This [text node][] is not [visible][] at a [viewport size][] of 640 by 512.
+
+```html
+<p style="display:none;">Last updated 2020/03/27 10:52pm</p>
+```
+
+#### Inapplicable Example 2
+
+This [text node][] has an SVG element as a [parent][].
+
+```html
+<svg>
+	<text x="0" y="15">I love SVG!</text>
+</svg>
+```
+
+#### Inapplicable Example 3
+
+This [text node][] has an [ancestor][] with `overflow: auto`.
```

Comment:
Comparing with Passed Example 4, this one is inapplicable due to "no ancestor with `overflow: hidden` or `clip`.
Both this one and Passed Example 4 have an ancestor with `overflow: auto`, it is not the reason why the example is inapplicable.

Reply:
I don't understand this comment. This is inapplicable because it has no ancestor with `overflow: hidden`, passed example 4 does.

Reply:
> This is inapplicable because it has no ancestor with `overflow: hidden`, passed example 4 does.

Yes, precisely.
The description here only tells about having "an ancestor with `overflow: auto`", suggesting that this is the reason why the example is inapplicable.
As you point out, the reason why this example is inapplicable is that "it has no ancestor with `overflow: hidden`" (or `clip` to keep the applicability wording). This is what the example description should state.
```suggestion
This [text node][] has no [ancestor][] with `overflow: hidden` or `clip`.
```